doc/languages-frameworks/python.section.md

```diff
@@ -280,30 +280,29 @@ order to build [`datashape`](https://github.com/blaze/datashape).
 ```nix
 { # ...
```

Comment:
This line doesn't make sense.

Reply:
Correct, not anymore. They existed because the examples represented attributes in an attribute set.
That is one thing that should be checked, what do the examples here represent.

Reply:
> That is one thing that should be checked, what do the examples here represent.

Sounds like perfect thinking to me.

Reply:
Note that the actual description above this is no longer fully reflects our situation
```
Our example, `toolz`, does not have any dependencies on other Python packages or
system libraries. According to the manual, `buildPythonPackage` uses the
arguments `buildInputs` and `propagatedBuildInputs` to specify dependencies. If
something is exclusively a build-time dependency, then the dependency should be
included as a `buildInput`, but if it is (also) a runtime dependency, then it
should be added to `propagatedBuildInputs`.
```